### PR TITLE
Django1.7.11

### PR DESCRIPTION
**Pull request by stxphcodes**
**Upgrade from Django1.6.11 to Django 1.7.11**

Documentation of backward incompatible changes, deprecations and removed functions as it applies for the eHB for can be viewed here: https://docs.google.com/spreadsheets/d/1SX6hwyCzQ_JOkGt29BBLLelcEvipQuMLR77MBtlE9wk/edit#gid=0 

**Major changes include:**
- removal of south to use Django's framework to migrate models 
- serialization of custom fields: EncryptCharField and EncryptDateField
- Redirection of admin login to a different url, added the new url pattern to middleware.py. 

**Instructions for upgrading to Django 1.7.11**
1. Pip install -r requirements.txt 
2. Pip uninstall south 
3. Run pip list and make sure that its Django 1.7.11 and south is not listed
4. Delete from ehb_service/apps/core/migrations all files except __init\__.py
5. Run python bin/manage.py migrate -> it should pick up that models have changed
6. Run python bin/manage.py makemigrations   -> new migration file should have been created '0001_inital.py'
7. Run python bin/manage.py migrate

### CODE REVIEW

File: fixtures/test_data.json [hunk 1]
```diff
@@ -935,13 +926,5 @@
       "content_type": 16
     }
   }
-  {
-    "pk":1,
-    "model": "core.PedigreeRelationshipRole",
```
Comment (seg1129) [inline fixtures/test_data.json]: Why was this fixture removed?

File: requirements.txt [hunk 2]
```diff
@@ -5,8 +5,7 @@
 #     pip install -U -r requirements.txt
 
 # Core requirements
-Django>=1.6,<1.7
```
Comment (seg1129) [inline requirements.txt]: These changes should also be reflected in the Dockerfile

File: ehb_service/conf/base.py [hunk 3]
```diff
@@ -20,7 +20,7 @@
     'django.contrib.staticfiles',
     # third-party
     'restlib2',
-    'south',
+    #'south',
```
Comment (seg1129) [inline ehb_service/conf/base.py]: I would just remove south instead of commenting it out

File: ehb_service/apps/core/migrations/0001_initial.py [hunk 4]
```diff
@@ -1,268 +1,282 @@
 # -*- coding: utf-8 -*-
```
Comment (seg1129) [inline ehb_service/apps/core/migrations/0001_initial.py]: for file ebb_service/apps/core/migrations/0001_initial.py can you change 'initial' to indicate this is a product of migrating to Django 1.7 and removing south? I just want to make sure when we look at this again - that we know that this is not the original migration, but the migration for removing south. Can you replace 'initial' with something short and sweet?

**Review by seg1129 — CHANGES_REQUESTED** [review]
Excellent work! I just have a few small changes

File: ehb_service/apps/core/migrations/0001_initial.py (hunk 4 above)
Comment (szh2425) [reply ehb_service/apps/core/migrations/0001_initial.py]: renamed to: django17Generated_initial b9aefce

File: fixtures/test_data.json (hunk 1 above)
Comment (szh2425) [reply fixtures/test_data.json]: I wasn't able to find the model core.PedigreeRelationshipRole in ehb_service/apps/core/models/identities.py so this fixture was giving an error when we ran 'python bin/manage.py loaddata  fixtures/test_data.json'. We do, however, have these following models related to relationships: 
- core.PedigreeSubjectRelation
- core.Relationship  
- core.ExternalRecordRelationship -> I think this is closest to the PedigreeRelationshipRole?

**Review by seg1129 — APPROVED** [review]

**Pull request merged**